src/lib/Components/Consignments/Screens/Provenance.tsx

```diff
@@ -36,8 +33,8 @@ export default class Provenance extends React.Component<Props, State> {
 
   render() {
     return (
-      <ConsignmentBG>
-        <DoneButton onPress={this.doneTapped}>
+      <Theme>
```

Comment:
@ds300 / @kierangillen - have ya'll thought about any possible patterns for hoisting this up, similar to how we have `Boot` in Reaction?

Reply:
@damassi yeah  I've been bitten by this a handful of times 😞 . It's not a priority though, and it will probably fall naturally out of the work we'll be doing to move the navigation into JS land.

Reply:
@ds300 @kierangillen @ashfurrow - do y'all plan on creating a [technical plan](https://www.notion.so/artsy/Technical-Plans-f94b206fcec54cee8b4d864e67d5b70f) for this navigation work?

Reply:
Yes! We've already half way-there, it just wasn't in the technical plans section of Notion yet.